### PR TITLE
[CodeCamp2023-498] Translation into Chinese of `tracking_train_test_zh_cn.md`

### PR DESCRIPTION
**Pull request by zhangzhidaSunny**
translate  tracking_train_test_zh_cn.md

Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

[CodeCamp2023-498] Translation into Chinese of an English document. (4)

## Modification

translate tracking_train_test_zh_cn.md into chinese

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMPreTrain.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by hhaAndroid** [comment]
@zhangzhidaSunny Please fix the lint

File: docs/zh_cn/user_guides/tracking_train_test_zh_cn.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+# 学会训练和测试
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/tracking_train_test_zh_cn.md:1]:
```suggestion
# 学习训练和测试
```

File: docs/zh_cn/user_guides/tracking_train_test_zh_cn.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+## 训练
+
+本节将介绍如何在支持的数据集上训练现有模型。
+支持以下训练环境：
+
+- CPU
+- 单 GPU
+- 单节点 多 GPU
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/tracking_train_test_zh_cn.md:10]:
```suggestion
- 单节点多 GPU
```

File: docs/zh_cn/user_guides/tracking_train_test_zh_cn.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+  例如，8 个 GPU * 1 个图像/GPU 的学习率为 `lr=0.01`，16 个 GPU * 2 个图像/GPU 的学习率为 lr=0.04。
+- 在训练过程中，日志文件和检查点将保存到工作目录、
+  该目录由 CLI 参数 `--work-dir`指定。它默认使用 `./work_dirs/CONFIG_NAME`。
+- 如果需要混合精度训练，只需指定 CLI 参数 `--amp`。
+
+#### 1.在 CPU 上训练
+
+该模型默认放在 cuda 设备上。
+仅当没有 cuda 设备时，该模型才会放在 CPU 上。
+因此，如果要在 CPU 上训练模型，则需要先`export CUDA_VISIBLE_DEVICES=-1`以禁用 GPU 可见性。
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/tracking_train_test_zh_cn.md:30]:
```suggestion
因此，如果要在 CPU 上训练模型，则需要先 `export CUDA_VISIBLE_DEVICES=-1` 以禁用 GPU 可见性。
```

File: docs/zh_cn/user_guides/tracking_train_test_zh_cn.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+CUDA_VISIBLE_DEVICES=-1 python tools/train.py ${CONFIG_FILE} [optional arguments]
+```
+
+在 CPU 上训练 MOT 模型 QDTrack 的示例：
+
+```shell 脚本
+CUDA_VISIBLE_DEVICES=-1 python tools/train.py configs/qdtrack/qdtrack_faster-rcnn_r50_fpn_8xb2-4e_mot17halftrain_test-mot17halfval.py
+```
+
+#### 2. 在单GPU上训练
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/tracking_train_test_zh_cn.md:43]:
```suggestion
#### 2. 在单 GPU 上训练
```

File: docs/zh_cn/user_guides/tracking_train_test_zh_cn.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+
+如果您想在单GPU上训练模型, 您可以按照如下方法直接使用 `tools/train.py`.
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/tracking_train_test_zh_cn.md:45]:
```suggestion
如果您想在单 GPU 上训练模型, 您可以按照如下方法直接使用 `tools/train.py`.
```

File: docs/zh_cn/user_guides/tracking_train_test_zh_cn.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+```
+
+## 测试
+
+本节将介绍如何在支持的数据集上测试现有模型。
+支持以下测试环境：
+
+- CPU
+- 单 GPU
+- 单节点 多 GPU
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/tracking_train_test_zh_cn.md:135]:
```suggestion
- 单节点多 GPU
```

**Comment by hhaAndroid** [comment]
@zhangzhidaSunny Please fix the lint

**Review by hhaAndroid — APPROVED** [review]

**Comment by OpenMMLab-Assistant-004** [comment]
Hi @zhangzhidaSunny,

We'd like to express our appreciation for your valuable contributions to the mmdetection. Your efforts have significantly aided in enhancing the project's quality.
It is our pleasure to invite you to join our community thorugh Discord_Special Interest Group (SIG) channel. This is a great place to share your experiences, discuss ideas, and connect with other like-minded people. To become a part of the SIG channel, send a message to the moderator, OpenMMLab, briefly introduce yourself and mention your open-source contributions in the #introductions channel. Our team will gladly facilitate your entry. We eagerly await your presence. Please follow this link to join us: ​https://discord.gg/UjgXkPWNqA.

If you're on WeChat, we'd also love for you to join our community there. Just add our assistant using the WeChat ID: openmmlabwx. When sending the friend request, remember to include the remark "mmsig + Github ID".

Thanks again for your awesome contribution, and we're excited to have you as part of our community!

**Pull request merged**